docs/guides/integration-options-private-operator.md

```diff
@@ -1,25 +1,35 @@
 ---
-title: Private Operator Integrations
-description: Overview of UID2 Private Operator options.
+title: UID2 Integrations for Private Operators
+description: Information summary for Private Operators.
 hide_table_of_contents: false
-sidebar_position: 02
 ---
 
 import Link from '@docusaurus/Link';
 
-# Private Operator Integrations
+# UID2 Private Operator Integration Overview
 
-A Private Operator is a private instance of the UID2 <Link href="../ref-info/glossary-uid#gl-operator">Operator</Link>. This means that a specific entity hosts a private instance of the UID2 Operator, exclusively for their own use.
+Private Operators of UID2 send first-party <Link href="../ref-info/glossary-uid#gl-dii">directly identifying information (DII)</Link> to a secure environment for translation, and control the destinations for those identifiers. A participant that chooses to become a Private Operator can generate and manage UID2s, running a UID2 <Link href="../ref-info/glossary-uid#gl-operator">Operator</Link> service in a private environment.
```

Comment:
Not sure about this paragraph. It implies that the Private Operator sends the DII out - which it doesn't. Maybe:

Participants that host their own Private Operator send first-party directly identifying information (DII) to their own, local UID2 Operator service, running in a private environment.

Reply:
Good point thx. Done.

Reply:
This still says that Private Operators send first party data ...
Participants send their first party data to their own Private operators.

Reply:
Hi @thomasm-ttd I'm not seeing this. What I had is what you suggested:
Participants that host their own Private Operator send first-party directly identifying information (DII) to their own, local UID2 Operator service, running in a private environment.
Not sure how to address this.
I've updated slightly to say this:
send **their own** first-party directly identifying information (DII) to their own, local UID2 Operator service
Is that any better?